### DISCUSSION TITLE
Reservation flow

### DISCUSSION THREAD

**Discussion by micaiah-effiong** [chimdie/shorttiee#25] [General]
What is the flow for creating a reservation on a shortlet

**Comment by chimdie** [comment]
1. Click on the book now button
2. Accept the disclaimer
3. Select your reservation period and continue[Sheet 1]
4. Preview and comfirm reservation

**Reply by micaiah-effiong** [reply]
@chimdie, what happens after the user confirms the reservation?

**Reply by chimdie** [reply]
1. The host should receive a notification and accept/decline
2. But then we have to comfirm what happens on the host end - if the apartment booked is available at that time and any other case

**Reply by sammy-code98** [reply]
Yeah, on the host part there should be a status eg `available`, `open` etc that will always reflect on the user part. this should be covered under the `edit shortlet` flow

**Reply by sammy-code98** [reply]
On the host part, here is a proposed flow for reservation
When a user books a reservation, the host sees it under the `incoming reservation` table and has the option to either accept or decline.
- once accepted, we email the user with payment details(set by the host at the point of acceptance) and then they can both proceed the transaction from there.
- if the host declines, we still email the user, so they can be informed and not keep waiting.
